src/controllers/rewards/post.action.ts

```diff
@@ -11,6 +11,7 @@ export const postReward = async (req: Request, res: Response) => {
         req.body.isMembershipRequired,
         req.body.isClaimOnce,
         req.body.withdrawCondition,
+        req.body.withdrawUnlockDate || new Date(0),
```

Comment:
@valeriagrazzini This will set the date to `Thu Jan 01 1970 01:00:00 GMT+0100 (Central European Standard Time)` right? Is that what we want?